package.json

```diff
@@ -16,34 +16,34 @@
     "validate-yaml": ".npm-scripts/validate-yaml.sh",
     "validate-openapi-spec": ".npm-scripts/validate-openapi-spec.sh",
     "start": "node src",
-    "start-debug": "nodemon --inspect=0.0.0.0:9229 src",
+    "start-debug": "nodemon --trace-warnings --inspect=0.0.0.0:9229 src",
     "test": ".npm-scripts/test.sh",
     "test-coverage": "jest --coverage",
     "smoke-test": "test/smoke/run.sh"
   },
   "dependencies": {
-    "ajv": "^6.10.1",
+    "ajv": "^8.11.0",
     "bcryptjs": "^2.4.3",
-    "body-parser": "^1.19.0",
-    "bunyan": "^1.8.12",
+    "body-parser": "^1.20.0",
+    "bunyan": "^1.8.15",
     "bunyan-format": "^0.2.1",
     "compression": "^1.7.4",
     "cors": "^2.8.5",
-    "dotenv": "^7.0.0",
-    "express": "^4.17.1",
-    "express-openapi": "^4.6.4",
-    "fast-json-patch": "^2.1.0",
-    "js-yaml": "^3.13.1",
-    "mongodb": "^3.2.7",
-    "mqtt": "^2.18.8",
+    "dotenv": "^16.0.1",
+    "express": "^4.18.1",
+    "express-openapi": "^11.0.1",
+    "fast-json-patch": "^3.1.1",
+    "js-yaml": "^4.1.0",
+    "mongodb": "^4.6.0",
+    "mqtt": "^4.3.7",
     "openapi-schema-validation": "^0.4.2",
-    "uuid": "^3.3.2"
+    "uuid": "^8.3.2"
   },
   "devDependencies": {
-    "husky": "^1.3.1",
-    "jest": "^24.8.0",
-    "lint-staged": "^8.2.1",
-    "nodemon": "^1.19.1",
+    "husky": "^8.0.1",
+    "jest": "^28.1.0",
+    "lint-staged": "^12.4.2",
+    "nodemon": "^2.0.16",
     "xo": "^0.24.0"
```

Comment:
I'm excluding xo for now, since it caused many lint errors due to changed rule set.
I would also like to switch to prettier at some point.